portfolio/src/main/webapp/index.html

```diff
@@ -36,6 +36,17 @@ <h1>Megan Shi</h1>
         </a>
       </p>
     </div>
-    <div id="greeting-container"></div>
+    <p>Leave a Comment:</p>
+    <form action="/data" method="POST">
+      <p>Enter display name here:</p>
+      <textarea name="name"></textarea>
+      <br/>
```

Comment:
nit: `<br>` is preferred over `<br/>` (please do the same for the rest below)

https://google.github.io/styleguide/htmlcssguide.html#Document_Type
Although fine with HTML, do not close void elements i.e. write `<br>`, not `<br />`.